src/components/ArpSpoof/ArpSpoof.tsx

```diff
@@ -169,28 +173,36 @@ const ARPSpoofing = () => {
     };
 
     return (
-        <>
-            {!loadingModal && (
-                <InstallationModal
-                    isOpen={opened}
-                    setOpened={setOpened}
-                    feature_description={description_userguide}
-                    dependencies={dependencies}
-                ></InstallationModal>
-            )}
-            <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
-                {LoadingOverlayAndCancelButton(loading, pidGateway)}
-                {LoadingOverlayAndCancelButton(loading, pidTarget)}
-                <Stack>
-                    {UserGuide(title, description_userguide)}
-                    <TextInput label={"Target one IP address"} required {...form.getInputProps("ip1")} />
-                    <TextInput label={"Target two IP address"} required {...form.getInputProps("ip2")} />
-                    <Button type={"submit"}>Spoof</Button>
-                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                    <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-                </Stack>
-            </form>
-        </>
+        <RenderComponent
+            title={title}
+            description={description_userguide}
+            steps={steps}
+            tutorial={Tutorial}
+            sourceLink={SourceLink}
+        >
+            <>
+                {!loadingModal && (
+                    <InstallationModal
+                        isOpen={opened}
+                        setOpened={setOpened}
+                        feature_description={description_userguide}
+                        dependencies={dependencies}
+                    ></InstallationModal>
+                )}
+                <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
+                    {LoadingOverlayAndCancelButton(loading, pidGateway)}
+                    {LoadingOverlayAndCancelButton(loading, pidTarget)}
+                    <Stack>
+                        {UserGuide(title, description_userguide)}
+                        <TextInput label={"Target one IP address"} required {...form.getInputProps("ip1")} />
+                        <TextInput label={"Target two IP address"} required {...form.getInputProps("ip2")} />
+                        <Button type={"submit"}>Spoof</Button>
```

Comment:
The baseline convention is Start + Title. 
```suggestion
                        <Button type={"submit"}>Start {title}</Button>
```

Reply:
Hi @carrju7 

I meant the code should be `Start {title}` rather than Start Spoof. Using {title} can insert the title variable into the button.